src/server/scripts/Northrend/FrozenHalls/PitOfSaron/pit_of_saron.cpp

```diff
@@ -1130,7 +1130,7 @@ class npc_pos_freed_slave : public CreatureScript
             me->LoadCreaturesAddon(true);
             me->SetLootRecipient(nullptr);
             me->ResetPlayerDamageReq();
-            me->SetLastDamagedTime(0);
+            me->UpdateLeashExtensionTime();
```

Comment:
Should be `ClearLastLeashExtensionTimePtr()`, right?

Reply:
Already fixed here: https://github.com/azerothcore/azerothcore-wotlk/commit/dd2167b00b138e97ff71cf04b54e2de147bb7089#diff-dd6621582e72d1647a24584f4b4ea94f550aea8ed7fe54a87535a2fd3a916d94R1133